plugins/basketball-scoreboard/manager.py

```
@@ -931,11 +940,39 @@ def _get_current_manager(self):
+                checker = FavoriteTeamCheck(self.logger, FAVORITE_CHECK_LEAGUES)
+                self._favorite_check = checker
+            for league in FAVORITE_CHECK_LEAGUES:
+                if not getattr(self, "{}_enabled".format(league), False):
+                    continue
+                for mode in ("live", "recent", "upcoming"):
+                    manager = getattr(self, "{}_{}".format(league, mode), None)
+                    favorites = getattr(manager, "favorite_teams", None)
+                    if favorites:
+                        checker.schedule(league, favorites)
+                        break
+        except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)